.github/workflows/ci.yml

```diff
@@ -0,0 +1,108 @@
+## (5b) Pin to commits, not tags
```

Comment:
What is this comment in reference to? (Same question for `release.yml`)

Reply:
See guidance document, section `Notes` (https://docs.google.com/document/d/16SQF6H7HNvKvZ4FDuqA5jRJsfWieTpaTMPmMZPRr7aQ/edit#heading=h.8xl8l37vtd7z), list item 2, about dependency versions. It may not apply here at all.

Reply:
Removed with commit dd3c282ee030db5090de1eeda91109eba4f1786a